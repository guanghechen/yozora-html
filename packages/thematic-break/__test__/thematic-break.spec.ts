import { ThematicBreakType } from '@yozora/ast'
import renderThematicBreak from '../src'

describe('snapshot', function () {
  it('basic', function () {
    expect(
      renderThematicBreak({
        type: ThematicBreakType,
      }),
    ).toMatchSnapshot()
  })
})
