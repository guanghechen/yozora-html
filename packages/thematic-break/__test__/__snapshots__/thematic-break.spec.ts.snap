// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot basic 1`] = `"<hr class=\\"yozora-thematic-break\\" />"`;
